docs/Frameworks/nextjs/1-Introducao.md

```diff
@@ -0,0 +1,51 @@
+<p align="center">
+  <a href="https://github.com/he4rt/4noobs" target="_blank">
+    <img src="../../../assets/global/header-4noobs.svg">
+  </a>
+</p>
+
+Aviso:
+Esta seção tem como objetivo ser um início para que você consiga começar a desenvolver aplicações Next.js
+
+Se você quiser se aprofundar ainda mais sobre cada um desses temas e sobre o funcionamento do Next.js em si, sinta-se livre para visitar o [NextJs4noobs](https://github.com/caioreix/NextJs4noobs)!
+
+# Next.js
+
+> Vá ponto a ponto ou vá direto ao que você está buscando! :
+
+1. Conceitos
```

Comment:
Como separou os arquivos agora os links não funcionam mais, não sei se é possível fazer um link no markdown para uma determinada posição de outro arquivo. Se não for possível apenas deixe um sumário com o link para o arquivo e em cada arquivo você replica a tabela de conteúdo do arquivo